api/models/usersChannels.js

```diff
@@ -637,7 +637,7 @@ const getUserPermissionsInChannel = (channelId: string, userId: string): Promise
     });
 };
 
-type UserIdAndChannelId = [string, string];
+type UserIdAndChannelId = [?string, string];
```

Comment:
Why is this first argument optional?

Reply:
Because the user might be logged out, at which point this will just return the default permissions :+1: